rust-runtime/aws-smithy-experimental/src/hyper_1_0.rs

```diff
@@ -437,19 +440,19 @@ where
     C::Error: Into<BoxError>,
 {
     fn call(&self, request: HttpRequest) -> HttpConnectorFuture {
-        let request = match request.try_into_http1x() {
+        let mut request = match request.try_into_http1x() {
             Ok(request) => request,
             Err(err) => {
                 return HttpConnectorFuture::ready(Err(ConnectorError::user(err.into())));
             }
         };
-        /*let capture_connection = capture_connection(&mut request);
+        let capture_connection = capture_connection(&mut request);
```

Comment:
Feel like this recently uncommented out chunk could use a comment for some context. Reading the [CaptureConnection docs](https://docs.rs/hyper-util/0.1.7/hyper_util/client/legacy/connect/struct.CaptureConnection.html) didn't quite clear it up for me.

Reply:
This is same as the hyper 0.14 implementation. `CaptureConnection` allows us to capture the actual connection used for the request (once established) and query it for information _or_ in this case poison it so it isn't used again.